kuma/wiki/tests/test_views.py

```diff
@@ -301,6 +301,14 @@ def _depth_test(depth, aught):
         json_obj = json.loads(resp.content)
         eq_(json_obj['subpages'][0]['title'], 'A Child')
 
+        # Test if we are serving an error json if document does not exist
+        no_doc_url = reverse('wiki.get_children', args=['nonexistentDocument'],
+            locale=settings.WIKI_DEFAULT_LANGUAGE)
+        resp = self.client.get(no_doc_url)
+        result = json.loads(resp.content)
+        aught = json.loads("""{"error": "Document does not exist."}""")
```

Comment:
No need to json.loads the thing that you expect, just use `{'error': 'Document does not exist.'}`.

Also, what's an aught? I know [oods](http://en.wikipedia.org/wiki/Ood), are they related?